### ISSUE TITLE
Class Prediction Softmax vs Sigmoid

### ISSUE THREAD

**Issue by developer0hye** [yjh0410/pytorch-yolo-v2#13]
Have you trained and tested your model using Sigmoid function instead of Softmax function?

In the YOLOv3 paper, the author mentioned that sigmoid function is used to predict multi-class.
Actually, I compared the mAP of 50 epoch trained 2 models with sigmoid and softmax and I observed that the model with softmax has higher mAP. But, I think 50 epochs trained model is not sufficient to compare the performance.

**Comment by yjh0410** [comment]
Sorry, I didn't as I had no need for multi-class.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Yonghye Kwon"<notifications@github.com&gt;;
发送时间:&nbsp;2020年3月16日(星期一) 下午4:23
收件人:&nbsp;"yjh0410/pytorch-yolo-v2"<pytorch-yolo-v2@noreply.github.com&gt;;
抄送:&nbsp;"Subscribed"<subscribed@noreply.github.com&gt;;
主题:&nbsp;[yjh0410/pytorch-yolo-v2] Class Prediction Softmax vs Sigmoid (#13)





 
Have you trained and tested your model using Sigmoid function instead of Softmax function?
 
In the YOLOv3 paper, the author mentioned that sigmoid function is used to predict multi-class.
 Actually, I compared the mAP of 50 epoch trained 2 models with sigmoid and softmax.

**Comment by developer0hye** [comment]
@yjh0410 
Thank you!
